app/controllers/devise/devise_authy_controller.rb

```diff
@@ -25,11 +25,9 @@ class Devise::DeviseAuthyController < DeviseController
 
   include Devise::Controllers::Helpers
 
+  # The verify_authy endpoints are for verification on login. Verification after registration
+  # is handled by the verify_installation methods.
```

Comment:
^^ very helpful comment TY for that

Reply:
I assume I was getting confused by it when I wrote it :laughing: